result-async/src/main/scala-2.13+/uscala/concurrent/result/AsyncResultImplicits.scala

```diff
@@ -0,0 +1,16 @@
+package uscala.concurrent.result
+
+import scala.collection.BuildFrom
+import scala.collection.mutable
+import scala.concurrent.ExecutionContext
+
+trait AsyncResultImplicits {
```

Comment:
The primary changes that effect us are the same over and over again:

* `CanBuildFrom` is out, `BuildFrom` is in.
* `Traversable` is out`, `Iterable` is the top now - so you are expected to use `IterableOnce` instead of `TraversableOnce` everywhere.
* You should use `BuildFrom#newBuilder` not `BuildFrom#apply`, which is deprecated.